src/components/tx/TxData.js

```diff
@@ -67,22 +67,41 @@ class TxData extends React.Component {
         label: 'Funds neighbors',
         ...this.baseItemGraph
       }
-    ]
+    ],
+    ncDeserializer: null,
   };
 
   // Array of token uid that was already found to show the symbol
   tokensFound = [];
 
-  componentDidMount = () => {
-    this.calculateTokens();
+  componentDidMount = async () => {
+    await this.handleDataFetch();
   }
```

Comment:
Since this is async now, is there any issues with the rendering?
Should we add a loading screen that goes away when this method finishes running?

Reply:
Makes sense. Added the loading following the UI we use in this screen: https://github.com/HathorNetwork/hathor-explorer/pull/253/commits/39fdb9a17cf350071ae0dfbfa420d710cf118ee9